src/test/java/com/google/firebase/testing/TestResponseInterceptor.java

```diff
@@ -41,8 +41,4 @@ public HttpResponse getResponse() {
     }
     return Iterables.getLast(responses);
```

Comment:
`responses` can now just be a single HttpResponse.

Reply:
Done